Content.Shared/_CM14/Xenos/Pheromones/SharedXenoPheromonesSystem.cs

```diff
@@ -230,13 +243,30 @@ public override void Update(float frameTime)
                 Dirty(uid, pheromones);
             }
 
-            _receivers.Clear();
-            _entityLookup.GetEntitiesInRange(xform.Coordinates, pheromones.PheromonesRange, _receivers);
```

Comment:
Okay so I wanted to add like a "NextUpdate" field but it looks like there's already one for plasma so we could just hook into that if that's okay smug?

Reply:
Sur